tests/schema.sql

```diff
@@ -50,6 +50,7 @@ CREATE TABLE `typemix` (
    `c_bit` bit(7) DEFAULT NULL,
    `c_int` int NOT NULL,
    `c_tinyint` tinyint NOT NULL,
+   `c_nullable_tinyint` tinyint,
```

Comment:
please also update the pgsql schema

Reply:
PostgreSQL doesn't have an equivalent of `tinyint`, so I'm not sure what to do here.

Reply:
for pgsql compat it would be better to introduce a different nullable `int` type as `tinyint` then.
at best one which is compatible between mysql and pgsql.

I think we already do some similar things in other tests, to keep them compatible between mysql and pgsql.
just have a look at the [MysqlIntegerRanges ](https://github.com/staabm/phpstan-dba/blob/5d759ac7c57fba21d31284937709ba923f4f8b51/src/Types/MysqlIntegerRanges.php)and [PgsqlIntegerRanges ](https://github.com/staabm/phpstan-dba/blob/5d759ac7c57fba21d31284937709ba923f4f8b51/src/Types/PgsqlIntegerRanges.php)class and find the type which is backend by the same integer range.

that way we don't need to rewrite all tests which use tinyint when pgsql is supported for the SQL AST feature.

Reply:
Do we really want to try to shoehorn MySQL types into PostgreSQL tables? As I mentioned in https://github.com/staabm/phpstan-dba/pull/573#issuecomment-1471839303, there are some fundamental differences between the two that cannot be reconciled.

If you really want, I'll add a nullable smallint column and call it `c_nullable_tinyint`, but it's not going to help make the tests reusable since it will be a 2-byte range instead of a 1-byte range.

Reply:
ok. lets leave it for now. I created a separate issue with the thing I have in mind